girderformindlogger/utility/jsonld_expander.py

```diff
@@ -311,7 +315,7 @@ def delanguageTag(obj):
 
 
 def expandOneLevel(obj):
-    if obj==None:
+    if obj is None:
```

Comment:
Have you considered changing this to ```if not obj:``` ? I don't know all of the use cases here, but ```not obj``` will also catch empty objects whereas ```obj is None``` only catches Nonetype objects. Just curious.

Reply:
https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/256/commits/65d75e4e905fd910e00b54532c70c8f6d7200fab

Reply:
https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/256#issuecomment-571295712